plugins/acss-kit/commands/kit-sync.md

```
@@ -0,0 +1,48 @@
+description: Bulk-install every acss-kit component, foundation, and starter theme into a project, tracked by .acss-kit/manifest.json for safe future updates
+argument-hint: [--target=<dir>] [--styles-dir=<dir>] [--seed=<hex>] [--skip-styles] [--dry-run]
+allowed-tools: Read, Glob, Grep, Write, Edit, Bash, AskUserQuestion
+---
+
+# /kit-sync
+
+Vendor every shipped acss-kit component, the `ui.tsx` foundation, and a starter OKLCH theme into your project in a single command. Records every file's normalized sha256 in `.acss-kit/manifest.json` so `/kit-update` can later re-copy unmodified files without clobbering your edits.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)